src/legacy/ui/public/chrome/directives/header_global_nav/components/header.tsx

```diff
@@ -88,10 +89,25 @@ function extendRecentlyAccessedHistoryItem(
   const href = relativeToAbsolute(chrome.addBasePath(recentlyAccessed.link));
   const navLink = navLinks.find(nl => href.startsWith(nl.subUrlBase));
 
+  let titleAndAriaLabel = recentlyAccessed.label;
+  if (navLink) {
+    const objectTypeForAriaAppendix = navLink ? navLink.title : '';
+    const translateKey = navLink.id.replace('kibana:', '');
+    const translateId = `common.ui.recentLinks.${translateKey}.ScreenReaderLabel`;
```

Comment:
@rockfield  Not sure that this is a valid case for our localization tool. Id should be static string. @maryia-lapata  please correct me if I'm wrong. 

I think we can just a remove ${translateKey} from id 
` titleAndAriaLabel = i18n.translate('common.ui.recentLinks.screenReaderLabel', {`

Reply:
@alexwizp there are 8 different links, -- that's why, unfortunately, it is impossible to use the same key as it guides to interconnections between different translations.

I guess that it is valid way to pass non-static ID, -- @maryia-lapata please correct me whether it is not correct ^_^